src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -3,85 +3,63 @@ title: Proof-of-stake (PoS)
 description: An explanation of the proof-of-stake consensus protocol and its role in Ethereum.
 lang: en
 sidebar: true
-incomplete: true
+incomplete: false
 ---
 
-Ethereum is moving to a consensus mechanism called proof-of-stake (PoS) from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/). This was always the plan as it's a key part in the community's strategy to scale Ethereum via [upgrades](/upgrades/). However getting PoS right is a big technical challenge and not as straightforward as using PoW to reach consensus across the network.
+Ethereum is moving to a consensus mechanism called proof-of-stake (PoS) from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/pow/). This was always the plan because PoS is thought to be more secure than PoW, uses drastically less energy, and enables new scaling solutions to be implemented. However, proof-of-stake is also more complex than proof-of-work. Refining the proof-of-stake mechanism has taken years of research and development, and the challenge now is to implement it on the live Ethereum network - a process known as ["The Merge"](/upgrades/merge/).
```

Comment:
Perhaps out of the scope of this PR (since this is already how the page is currently) but I wonder if it'd be best to first have a quick intro on what a consensus mechanism is & what PoS is before getting into how it relates specifically to the Ethereum roadmap?

Reply:
Just thinking that people often land on this page when searching "proof of stake". Many people may be looking for a general explanation before diving into the specifics on Ethereum.